pyanaconda/packaging/yumpayload.py

```diff
@@ -1165,6 +1165,24 @@ def groupDescription(self, groupid):
 
             return (group.ui_name, group.ui_description)
 
+    def groupId(self, group_name):
+        """Translate group name to group ID.
+
+        :param group_name: Valid identifier for group specification.
+        :returns: Group ID or empty string if nothing is found.
+        :raise NoSuchGroup: If group_name doesn't exists.
+        """
+        groups = self._yumGroups
+        if not groups:
+            return ""
```

Comment:
I want to have the *same* behavior as `groupDescription()` method has, this method is returning `(groupid, groupid)`.

It would be unexpected behavior when one method is returning something and second similar method below raising an exception.

Also catching an exception takes more performance than rewriting value in a dictionary.